Gemfile

```
@@ -20,7 +20,10 @@ group :development, :test do
   gem 'brakeman', '~> 6.0', require: false
   gem 'debug', platforms: %i[mri mingw x64_mingw]
   gem 'faker', '~> 3.2'
-  gem 'katakata_irb', '~> 0.1.9', require: false
+
+  github 'tompng/katakata_irb' do
+    gem 'katakata_irb', require: false
+  end
```

gem の引数としても指定できるので、このようにもかけますよ。
```suggestion
  gem 'katakata_irb', github: 'tompng/katakata_irb', require: false
```

---

おー、こちらのほうがシンプルでいいですね。調査不足でした。修正します。